docs/components.md

```diff
@@ -55,6 +55,18 @@ Independent, small visual UI elements to use in applications.
 * [service-telemetry](https://github.com/mozilla-mobile/android-components/tree/master/components/service/telemetry) - A generic library for sending telemetry pings from Android applications to Mozilla's telemetry service.
 * [service-glean](https://github.com/mozilla-mobile/android-components/tree/master/components/service/glean) - A generic library for sending telemetry pings from Android applications to Mozilla's telemetry service (eventually replacing `service-telemetry`).
 
+## Samples
+
+* [Browser (simple)](https://github.com/mozilla-mobile/android-components/blob/master/samples/browser) - very basic browser implementation based on the browser components.
+* [Browser (full-featured)](https://github.com/mozilla-mobile/reference-browser) - advanced browser implementation based on the browser components.
+* [Crash](https://github.com/mozilla-mobile/android-components/blob/master/samples/crash) - sample integration for the [lib-crash](https://github.com/mozilla-mobile/android-components/blob/master/components/lib/crash/README.md) component.
```

Comment:
Please capitalize the first word of the description to be consistent with the rest of the page.
```suggestion
* [Crash](https://github.com/mozilla-mobile/android-components/blob/master/samples/crash) - Sample integration for the [lib-crash](https://github.com/mozilla-mobile/android-components/blob/master/components/lib/crash/README.md) component.
```

Reply:
Good point.